lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+let rateTable = {};
```

Comment:
Is there a reason why this variable is set globally in the file as opposed to just within the `register` function?